src/node_crypto.h

```diff
@@ -65,6 +65,9 @@ extern int VerifyCallback(int preverify_ok, X509_STORE_CTX* ctx);
 
 extern X509_STORE* root_cert_store;
 
+extern void UseExtraCaCerts(const char* file);
```

Comment:
Tiny nit but the parameter is called `file` here in the header file but `fname` in the source file.